e2e/with-php-level/expected-output.diff

```diff
@@ -0,0 +1,18 @@
+1 file with changes
+===================
+
+1) src/ldap_test.php:0
+
+    ---------- begin diff ----------
+@@ @@
+ <?php
+
+-ldap_first_attribute($ldap, $entry, $more);
++ldap_first_attribute($ldap, $entry);
+    ----------- end diff -----------
+
+Applied rules:
+ * RemoveExtraParametersRector
```

Comment:
it should show `RemoveFuncCallArgRector` per php 5.2 config set, but it shows `RemoveExtraParametersRector` on php 7.1 set, it probably due to `array_merge()` config so configured config is late instead of added, that's seems needs separate PR for that.